seqr-gke/main.tf

```diff
@@ -35,9 +36,9 @@ resource "google_container_cluster" "cluster" {
 
   cluster_autoscaling {
     enabled = "false"
+    autoscaling_profile = "OPTIMIZE_UTILIZATION"
```

Comment:
The `enabled = "false"` is confusing, but I believe it's correct as is... as it is referring "node auto-provisioning", which, at the cluster level, means spinning up new node pools.